README.md

```diff
@@ -644,6 +644,9 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
 - `booleans` -- various optimizations for boolean context, for example `!!a
   ? b : c → a ? b : c`
 
+- `typeofs` -- `typeof foo == "undefined"` → `foo === void 0`
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2201/files#diff-04c6e90faac2675aa89e2176d2eec7d8R647